helix-term/src/commands.rs

```diff
@@ -1134,6 +1135,16 @@ fn goto_file_impl(cx: &mut Context, action: Action) {
     for sel in paths {
         let p = sel.trim();
         if !p.is_empty() {
+            if let Ok(url) = Url::parse(p) {
+                if let Err(e) = open::that(url.as_str()) {
```

Comment:
We should check for `file://` URLs and open them inside helix

Reply:
I wasn't sure about this part, for example if I have a link to a pdf file opening it in browser might make more sense. Anyway, changed the behaviour based on the suggestion to open `file://` urls in helix, happy for any additional suggestions.

Reply:
We already have a known list of files helix cannot open that we ignore in the filepicker. We could use that list to auto detect the most common files to open externally